### PR TITLE
[Semi-Modular?] More narcotics

### PR DESCRIPTION
**Pull request by linnpap**
## About The Pull Request
Hey, the Big Man's back! W W W, dot, the kid from Brooklyn, dot com.
image

## Why It's Good For The Game
Because the Big Man said so.

## Changelog
:cl:
add: Added quaaludes
add: Added cocaine
add: Added crack, along with crack pipes
add: Added opium and heroin
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
The big man would walk twenny miles for a QUAALUDE lemme tell ya

**Comment by Vizoplez** [comment]
`file ./modular_skyrat/modules/morenarcotics/chemistry/reagents/cocaine.dm is missing a trailing newline
file ./modular_skyrat/modules/morenarcotics/chemistry/reagents/quaalude.dm is missing a trailing newline
file ./modular_skyrat/modules/morenarcotics/chemistry/recipes/quaalude.dm is missing a trailing newline
file ./modular_skyrat/modules/morenarcotics/items/crackpipe.dm is missing a trailing newline`


Add newlines and this should be good.

**Comment by linnpap** [comment]
> 
> 
> The big man would walk twenny miles for a QUAALUDE lemme tell ya

BRING BACK THE FUCKING QUAALUUUDEEEESSSS

**Comment by Draganfrukts** [comment]
Add heroin from Opium and make it black tar heroin if you get it low purity, no balls

**Comment by Nerev4r** [comment]
Humble (actual) suggestion, but please give crack bricks export value (maybe if you hack the console like how you can export organs?) I'd like to see some cartel RP or something.

**Comment by gummardgumphrey** [comment]
THE FUCKIN' LUUDES

**Comment by Sababall** [comment]
Will we be able to check out how pure the cocaine is by putting it on our tongues?

**Comment by Nerev4r** [comment]
'If you eat harvested coca it should reduce drowsiness.'

**Comment by linnpap** [comment]
i think that somebody is going to die

**Comment by linnpap** [comment]
should be ready to be test merged, ill probably start working on heroin tomorrow

File: code/modules/reagents/chemistry/reagents/drug_reagents.dm
Comment (Gandalf2k15) [inline code/modules/reagents/chemistry/reagents/drug_reagents.dm]: Modular comments

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by hugmeorelse** [comment]
Friendly reminder to stay on topic. The Github is for technical analysis and code review. A joke here and there is fine, as long as it's good natured. I am not anal enough for that.

**Comment by thestubborn** [comment]
jessie where is the cocainer

File: modular_skyrat/modules/morenarcotics/chemistry/reagents/cocaine.dm
Comment (tf-4) [inline modular_skyrat/modules/morenarcotics/chemistry/reagents/cocaine.dm]: Use actual variable names, not random letters

**Comment by Gandalf2k15** [comment]
Once this is ready for merge, tell me

**Comment by linnpap** [comment]
> 
> 
> Once this is ready for merge, tell me

honestly i think it should be ready to be merged now

**Comment by linnpap** [comment]
actually hold up export values are fucked

**Pull request merged**
